app/controllers/owners/companies_controller.rb

```diff
@@ -28,6 +28,16 @@ def show
       render json: company
     end
 
+    def stats
+      company = Company.find(params[:company_id])
+
+      stats = company.areas.map do |a|
+        { area_name: a.name, checkin_count: a.tickets.open.count }
```

Comment:
```suggestion
      stats = company.areas.map do |area|
        { area_name: area.name, checkin_count: area.tickets.open.count }
```
I really prefer non-single-char variable names :smile: